examples/search/README.md

```diff
@@ -0,0 +1,21 @@
+# Search example
+
+A simple search app using tinyhttp.
+
+## Setup
+
+### Install dependencies
```

Comment:
there are no extra steps except installing dependencies so this subheading can be removed